spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationExecution.scala

```diff
@@ -77,6 +77,10 @@ trait StandardizationExecution extends CommonJobExecution {
     // Add the raw format of the input file(s) to Atum's metadata
     Atum.setAdditionalInfo("raw_format" -> cmd.rawFormat)
 
+    // Add Dataset properties marked with putIntoInfoFile=true
+    val dataForInfoFile: Map[String, String] = dao.getDatasetPropertiesForInfoFile(cmd.datasetName, cmd.datasetVersion)
+    addCustomDataToInfoFile(keyPrefix = "std_ds_", dataForInfoFile)
```

Comment:
I saw that the metadata fields are always prefixed somehow to result in a less messy list. So I both followed suit and thought it would be a good idea.

Do you suggest removing it completely or provide a configuration for this (with the default of an empty prefix)?

Reply:
I think the properties are Dataset globals. Doesn't really care about Std and Conf phases.
I am not even sure ATUM has a notion of them. So that why it feels forced. ATUM is checkpoint oriented, I would say. (but to add the properties to every checkpoint seems as overreach.).

Reply:
Also I think it should be part of it's own subpath (`properties`). Probably configurable.. 🤔

Reply:
I have made the prefix configurable under `control.info.dataset.properties.prefix`, default = "". You can now set the prefix to anything, even `properties.`, but this result in a lower hierarchy level in the `_INFO` file json. 
(This is how ATUM deals with the `controlMeasure.metadata.additionalInfo` structure. We would have to add the support for this to ATUM first. Feel free to file a feature request in ATUM if you think we should.)